app/src/main/java/io/apicurio/registry/storage/impl/sql/AbstractSqlRegistryStorage.java

```diff
@@ -687,7 +679,6 @@ private void insertReferences(Handle handle, Long contentId, List<ArtifactRefere
     }
 
     @Override
-    @Transactional
     public List<String> deleteArtifact(String groupId, String artifactId)
```

Comment:
We should use the GA/GAV value types everywhere to reduce accidental errors, since String is a String is a String. I'd like to do that for other identifiers in the future.

Reply:
This is out-of-scope for this PR though.